src/cljs/owlet_ui/components/search_bar.cljs

```diff
@@ -29,7 +29,8 @@
           (into []
                 (take @suggestion-count
                       (for [n (distinct search-collections)
-                            :when (re-find (re-pattern (str "(?i)" s)) n)]
+                            :when (try (re-find (re-pattern (str "(?i)" s)) n)
```

Comment:
hey @ZadenRB so this works but i read [here](https://clojuredocs.org/clojure.core/try#example-542692d6c026201cdc3270d5) that we should use:
`(catch js/Object e)` instead. Lmk what you think.